app/src/main/res/layout/fragment_onboarding_page.xml

```diff
@@ -43,6 +43,7 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/medium_margin_size"
         android:layout_marginEnd="@dimen/medium_margin_size"
+        android:alpha="0.7"
```

Comment:
Since guidelines aren't visible to the user, I assume you misplaced this field here :P